pkgs/applications/window-managers/sway/sway-config-no-nix-store-references.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/config.in b/config.in
+--- a/config.in
++++ b/config.in
+@@ -21,8 +21,8 @@ set $menu dmenu_path | dmenu | xargs swaymsg exec
+ 
+ ### Output configuration
+ #
+-# Default wallpaper (more resolutions are available in @datadir@/backgrounds/sway/)
+-output * bg @datadir@/backgrounds/sway/Sway_Wallpaper_Blue_1920x1080.png fill
++# Default wallpaper
++#output * bg ~/.config/sway/backgrounds/Sway_Wallpaper_Blue_1920x1080.png fill
```

Comment:
Why is this done? The files are installed in the store path, right?

Reply:
That's because most users (including me) will probably choose the default workflow from the wiki (https://github.com/swaywm/sway/wiki#configuration) which I also referred to in the documentation of the `enable` option:

> The suggested location for the sway configuration file is `~/.config/sway/config`. To begin configuring sway, create this directory and copy the default config.
> ```
> mkdir -p ~/.config/sway
> cp /etc/sway/config ~/.config/sway/
> $EDITOR ~/.config/sway/config
> ```
> Read the default config - it has comments that explain what each option does. Read `man 5 sway` for more information about each config command.

But then their copy won't get updated and the `/nix/store/` paths will eventually getting garbage collected. And I also stopped installing the default wallpapers into `/nix/store/` but we could revert that.

`~/.config/sway/backgrounds/Sway_Wallpaper_Blue_1920x1080.png` is not optimal but I kept it as an example.

An alternative would be to install the default wallpapers e.g. into `/run/current-system/sw/share/sway/wallpapers` if we want to keep them.